tests/test_pretty_midi.py

```diff
@@ -213,7 +213,7 @@ def simple():
     # Original tempo change times: [0, 6, 8.1, 8.3, 9.3]
     # Plus tempo changes at each of new_times which are not collapsed
     # Plus tempo change at 0s by default
-    expected_times = [0., 5., 6., 8.5,
+    expected_times = [0., 5., 6, 8.5,
```

Comment:
Nitpick: Can you retain the decimal in this 6?

Reply:
Sure. Good catch.